cluster/ansible/playbooks/roles/master/templates/etcd.jsonnet

```diff
@@ -0,0 +1,70 @@
+function(cfg)
+  {
+    apiVersion: "v1",
+    kind: "Pod",
+    metadata: {
+      name: "etcd-server",
+      namespace: "kube-system",
+    },
+    spec: {
+      hostNetwork: true,
+      containers: [
+        {
+          name: "etcd-container",
+          image: "gcr.io/google_containers/etcd:2.2.1",
+          resources: {
+            requests: {
+              cpu: "200m",
+            },
+          },
+          command: [
+            "/bin/sh",
+            "-c",
+            |||
+              /usr/local/bin/etcd \
+                --listen-peer-urls http://127.0.0.1:2380 \
+                --addr 127.0.0.1:4001 \
```

Comment:
nit: In etcd v2 the default client port moved to 2379 (4001 will be deprecated): https://github.com/coreos/etcd/blob/master/Documentation/v2/README.md#etcd-tcp-ports